spec/end-to-end.coffee

```diff
@@ -1442,4 +1467,137 @@ describe 'End - to - End', ->
         margin-right: 9px;
       }
       
-      ###
\ No newline at end of file
+      ###
+
+
+    describe 'implied property values', ->
+
+      it 'should compute values', (done) ->
+        container.innerHTML = """
+          <div id="header"></div>
+
+          <style type="text/gss">
+            @h |[#header]| in(::window);
+            [header-center-x] == #header[center-x];
+          </style>
+        """
+
+        windowWidth = window.innerWidth - GSS.get.scrollbarWidth()
```

Comment:
On my new branch these all fail because `GSS.get.scrollbarWidth()` is `undefined`.